### ISSUE TITLE
New Rule: Description Validation

### ISSUE THREAD

**Issue by ntwb** [tclindner/npm-package-json-lint#89]
A rule to validate description format.

It was mentioned https://github.com/WordPress/gutenberg/pull/8121#discussion_r204443577 that it would be great to have a rule to ensure that a description ended with a `.` period:

e.g. `My package description.` rather than `My package description`

Another possability would be to ensure that the description started with a capital letter:

e.g. `My package description.` rather than `my package description`

**Comment by tclindner** [comment]
Thanks, @ntwb! Do you have any thoughts on whether you foresee other validations of the description in the future? I'm debating if we should create a rule called `description-format` that only takes a severity level or if we should allow it to have its own options like `requireCapitalFirstLetter` and `requireEndingPeriod`. What do you think?

**Comment by ntwb** [comment]
Right now I can't think of any extra validations that may be required in the future.

Having the options would somewhat future proof that though might it?

**Comment by tclindner** [comment]
Yeah, definitely. I'll get started on this soon and let you know when it is ready! Thanks, @ntwb
